__init__.py

```diff
@@ -212,7 +212,7 @@ def speak_interval(self, start, stop, max_results=None):
         events = eventsResult.get('items', [])
         if not events:
             LOG.debug(start)
-            d = datetime.strptime(start.split('.')[0], '%Y-%m-%dT%H:%M:%SZ')
+            d = datetime.strptime(start.split('.')[0] + 'Z', '%Y-%m-%dT%H:%M:%SZ')
```

Comment:
Just started to review these updates. This extra 'Z' causes issues for me, is it necessary for it to work for you?

Reply:
Hmm, yes it was, it caused issues for me when it was missing, because the string passed into the function didn't include it when called from `get_left_today` 🤔 Is it not the same for you?

Reply:
Ok I think I see the issue The split on '.' that I had doesn't quite work here since the Z you apply is after the .xxxx. I'll merge this manually and add a proper fix

Reply:
Oh yes, that seems to be the problem, great!